packages/tests/utils/BasePage.ts

```diff
@@ -81,8 +81,8 @@ export class BasePage<T extends HTMLElement = HTMLElement> {
     return this;
   }
 
-  makeSnapshot(name?: string, opts?: Parameters<typeof cy.percySnapshot>[1]) {
-    cy.percySnapshot(name, opts);
+  makeSnapshot(config?: Parameters<typeof cy.matchImage>[1]) {
```

Comment:
Suggestions: Shouldn't this be:
```suggestion
  makeSnapshot(config?: Parameters<typeof cy.matchImage>[0]) {
```